docs/topics/OBST-wf-swarm-crud-introduction.adoc

```diff
@@ -0,0 +1,28 @@
+= Introduction
```

Comment:
Definitely will do. I'll create a common introduction segment for all implementations within the CRUD experience.